specs.md

```diff
@@ -0,0 +1,138 @@
+Statements for imperative proofs (SOLAR-style)
+====================
+
+```k
+requires "evm.k"
+
+module EVM-SPECS
+    imports ID
+    imports EVM
+
+    configuration
+      <kevm-specs>
+        <kevm/>
+        <commandVars> .Map </commandVars>
+      </kevm-specs>
+
+// Core specification scripting language
+// ======================================================================
+
```

Comment:
End the code block, have this be a proper level two header:

```
Core Specification Scripting Language
-------------------------------------
```

then start the next code block.